docs/learning/useful-resources/setup-prettier-and-vale.md

```diff
@@ -0,0 +1,30 @@
+# Setup Prettier and Vale for VS Code
+
+Everyone at Thoth Tech should be using both
+[Prettier](https://prettier.io/), as well as [Vale](https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server). These allow for Pull Requests to run smoothly and allow Thoth Tech to have a consistent format, free from grammar and spelling issues. 
+
+
+
+## Setting up Prettier
+To install Prettier do the following:
+1. In Visual Studio code go to `Extensions` page.
```

Comment:
Do we have methods for using Prettier without VSC? Maybe a requirements section or something similar would help